src/room.js

```diff
@@ -19,19 +25,28 @@ import "./components/virtual-gamepad-controls";
 import "./components/body-controller";
 import "./components/hand-controls2";
 import "./components/character-controller";
-import "./components/split-axis-events";
+import "./components/haptic-feedback";
 import "./components/networked-video-player";
 import "./components/offset-relative-to";
 import "./components/cached-gltf-model";
 import "./components/spawn-controller";
 import "./systems/personal-space-bubble";
 
-import registerNetworkScheams from "./network-schemas";
-import registerInputMappings from "./input-mappings";
 import { promptForName, getCookie, parseJwt } from "./utils";
-
-registerNetworkScheams();
-registerInputMappings();
+import registerNetworkSchemas from "./network-schemas";
+import { inGameActions, config } from "./input-mappings";
+
+AFRAME.registerInputBehaviour("vive_trackpad_dpad4", vive_trackpad_dpad4);
```

Comment:
There is now enough stuff here that I'd maybe move all this into input-mappings.js and export a `registerInputMappigns` function that sets up all this stuff, similar to what we do for network schemas.